### PR TITLE
Forbedring run terraform

### PR DESCRIPTION
**Pull request by EbiJon**
The developers are allowed now to write the branch name without the full path. 
Example: (master) instead of (refs/heads/master).
OPS To Ensure backwards compatibility both branch name and full path are accepted.

### CODE REVIEW

File: .github/workflows/run-terraform.yml [hunk 1]
```diff
@@ -117,6 +116,15 @@ env:
...
 
 jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Echo 
+        env:
+        
+          GITHUB_CONTEXT: ${{ toJSON(github) }}
+        run: echo "$GITHUB_CONTEXT"
```
Comment (anderssonw) [inline .github/workflows/run-terraform.yml:126]: husk å fjerne denne

File: .github/workflows/run-terraform.yml [hunk 2]
```diff
@@ -130,6 +138,10 @@ jobs:
           OVERRIDE=$WORKLOAD_IDENTITY_PROVIDER_OVERRIDE
           PROVIDER=${OVERRIDE:-$DEFAULT_WORKLOAD_IDENTITY}
           echo "WORKLOAD_IDENTITY_PROVIDER=$PROVIDER" >> $GITHUB_OUTPUT
+      - name: Echo github context 
+        env:
+          GITHUB_CONTEXT: ${{ toJSON(github) }}
+        run: echo "$GITHUB_CONTEXT"
```
Comment (anderssonw) [inline .github/workflows/run-terraform.yml:144]: og denne

**Review by anderssonw — CHANGES_REQUESTED** [review]

**Comment by anderssonw** [comment]
husk også å bruke squash and merge på denne, mye teste-commits :P

File: .github/workflows/run-terraform.yml [hunk 3]
```diff
@@ -131,6 +132,7 @@ jobs:
           PROVIDER=${OVERRIDE:-$DEFAULT_WORKLOAD_IDENTITY}
           echo "WORKLOAD_IDENTITY_PROVIDER=$PROVIDER" >> $GITHUB_OUTPUT
 
+
```
Comment (anderssonw) [inline .github/workflows/run-terraform.yml:135]: denne og

**Review by anderssonw — APPROVED** [review]

**Review by berg-thom — APPROVED** [review]

**Pull request merged**